File: gs.api/contracts/reseller/dto/registration/OrganizationSettings.cs
using System.Runtime.Serialization;

namespace gs.api.contracts.reseller.dto.registration
{
    [DataContract]
    public class OrganizationSettings
    {
        public OrganizationSettings(string ownerFirstName, string ownerLastName, string ownerPatronymic,
            string tradeMark, string fullName, string address, string phone, string inn, bool useVat)
        {
            OwnerFirstName = ownerFirstName;
            OwnerLastName = ownerLastName;
            OwnerPatronymic = ownerPatronymic;
            TradeMark = tradeMark;
            FullName = fullName;
            Address = address;
            Phone = phone;
            Inn = inn;
            UseVat = useVat;
        }

        [DataMember(IsRequired = false)]
        public string OwnerFirstName { get; set; }
        
        [DataMember(IsRequired = false)]
        public string OwnerLastName { get; set; }
        
        [DataMember(IsRequired = false)]
        public string OwnerPatronymic { get; set; }
        
        [DataMember(IsRequired = false)]
        public string TradeMark { get; set; }
        
        [DataMember(IsRequired = false)]
        public string FullName { get; set; }
        
        [DataMember(IsRequired = false)]
        public string Address { get; set; }
        
        [DataMember(IsRequired = false)]
        public string Phone { get; set; }
        
        [DataMember(IsRequired = false)]
        public string Inn { get; set; }
        
        [DataMember(IsRequired = false)]
        public bool UseVat { get; set; }
    }
}

File: gs.front/src/app/models/settings/organization-settings.ts
export class OrganizationSettings {
  constructor(ownerFirstName: string, ownerLastName: string, ownerPatronymic: string, tradeMark: string,
              fullName: string, address: string, phone: string, inn: string, useVat: boolean) {
    this.ownerFirstName = ownerFirstName;
    this.ownerLastName = ownerLastName;
    this.ownerPatronymic = ownerPatronymic;
    this.tradeMark = tradeMark;
    this.fullName = fullName;
    this.address = address;
    this.phone = phone;
    this.inn = inn;
    this.useVat = useVat;
  }

  ownerFirstName: string;
  ownerLastName: string;
  ownerPatronymic: string;
  tradeMark: string;
  fullName: string;
  address: string;
  phone: string;
  inn: string;
  useVat: boolean;

  public static Empty(): OrganizationSettings {
    return new OrganizationSettings('', '', '', '', '', '',
      '', '', false);
  }
}
